packages/core/src/api/formatConversions/formatConversions.ts

```diff
@@ -131,3 +135,31 @@ export async function markdownToBlocks<BSchema extends BlockSchema>(
 
   return HTMLToBlocks(htmlString.value as string, blockSchema, schema);
 }
+
+// Takes structured HTML and makes it comply with the HTML spec using the
+// `simplifyBlocks` plugin.
+// TODO: Remove classes?
+export async function cleanHTML(structuredHTMLString: string) {
```

Comment:
blocks should be able to specify their own "clean html", right? So this can never be a `string -> string` method?

Reply:
I think we need some more tests now that you mention it. Since right now we're only testing image custom blocks, everything works. This is because for structured HTML, we only need the `blockContent` div to recreate the block, anything inside is not relevant.

Therefore, for copy/paste, we can just put the HTML from the image block's `serialize` function inside the `blockContent` div instead of the output from `render`. But image blocks don't have inline content, which should be found somewhere inside the `blockContent` div.

So you're right, this might have to be changed. I think right now we should add unit tests for custom blocks with inline content (e.g. an alert block).

Reply:
I think this could also be the case for blocks without inline content, let's discuss